### PR TITLE
Updated Suricata Configuration

### PR DESCRIPTION
**Pull request by hariomch**
Upgraded Suricata to the latest release.
Updated Suricata configuration to v6.0 release and added multiple protocol logging support to it.
Issue: #1

### CODE REVIEW

File: config/suricata/etc/suricata.yaml [hunk 1]
```diff
@@ -39,44 +39,58 @@ vars:
...
   #decoder-events-prefix: "decoder.event"
   # Add stream events as stats.
   #stream-events: false
 
+# Configure the type of alert (and other) logging you would like.
 outputs:
+  # a line based alerts log similar to Snort's fast.log
   - fast:
-      enabled: no
+      enabled: yes
```
Comment (ManofWax) [inline config/suricata/etc/suricata.yaml]: What is it used for?

File: config/suricata/etc/suricata.yaml [hunk 2]
```diff
@@ -85,136 +99,444 @@ outputs:
...
+        - rdp
+        - nfs
+        - smb
+        - tftp
+        - ikev2
+        - dcerpc
+        - krb5
+        - snmp
+        - rfb
+        - sip
```
Comment (ManofWax) [inline config/suricata/etc/suricata.yaml:244]: Are these enabled?

File: docker-compose.yaml [hunk 3]
```diff
@@ -72,7 +72,7 @@ services:
   
     suricata:
         container_name: pcapmonkey_suricata
-        image: jasonish/suricata:6.0.0
+        image: jasonish/suricata:latest
```
Comment (ManofWax) [inline docker-compose.yaml]: I think we should keep pinning the container to a known release. It's much safer this way, and we should manually testing the suricata versione before updating it

**Review by ManofWax — CHANGES_REQUESTED** [review]

File: config/suricata/etc/suricata.yaml (hunk 1 above)
Comment (hariomch) [reply config/suricata/etc/suricata.yaml]: fast.log contains alerts consisting of a single line similar to Snort's fast.log. It comes enabled in the default configuration of Suricata. So basically the output generated by Suricata are written to the  ```fast.log``` along with ```eve.json``` file.

File: config/suricata/etc/suricata.yaml (hunk 2 above)
Comment (hariomch) [reply config/suricata/etc/suricata.yaml:244]: Yes, these protocols are enabled and Suricata logs their outputs to the ```eve.json```

**Review by ManofWax — APPROVED** [review]

**Pull request merged**